azure-pipelines.yml

```diff
@@ -210,6 +210,28 @@ stages:
               displayName: Sign packages
 
 - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  - stage: Create_BAR_ID_Tag
```

Comment:
Depending on if you care when this stage runs, you could add 

```
dependsOn: []
```
and then it would run in parallel with whatever the first stage is, and tag every non-public, non-pr build.

Reply:
I think where it is works, as this was Ricardo's recommendation. But that's a good idea in case we need it :)